packages/reference/src/components/LinkActions/LinkActions.tsx

```diff
@@ -4,7 +4,7 @@ import * as styles from './styles';
 import { EntityType, ContentType, ActionLabels } from '../../types';
 import { CreateEntryLinkButton } from '../CreateEntryLinkButton/CreateEntryLinkButton';
 
-interface LinkActionsProps {
+export interface LinkActionsProps {
```

Comment:
I think we should actually export this in the main `index.tsx` just like we expose the `CustomEntryCardProps` [here](https://github.com/contentful/field-editors/blob/fd1d9353d459f96f255430eba0f6b5217c30f973/packages/reference/src/index.tsx#L16)

As we're making this a public interface, we should also clean it up a bit. E.g. `multiple` is not consistent with most our public naming, e.g. `isMultipleLinks: boolean` would be a bit clearer.

A nice property to add (not sure if you need it though right now) would be one to indicate the current number of links to e.g. render different action buttons for an empty state link field editor.

Reply:
All of the things you mentioned are good points, I'd skip them for time being and once we see actual need for these "features" I'd happily come back and implement them.

Reply:
The first two points are not features, I think you need to export `LinkActionsProps` from the main package to be able to use it. Renaming `multiple` should be done for consistency/clarity.